packages/jest-expo/bin/jest.js

```diff
@@ -16,8 +16,8 @@ const process = require('process');
 //
 // If you need to run Jest with the JS debugger enabled, run Jest directly. It is usually under
 // node_modules/jest/bin/jest.js.
-const jestPackagePath = path.resolve(require.resolve('jest/package.json'), '..');
-const jestProgramPath = path.resolve(jestPackagePath, jestPackageJson.bin.jest);
```

Comment:
I never considered that going from `bin: {jest: 'path'}` to `bin: 'path'` (which works when the name of the bin matches the name of the package) would be breaking.

This could fetch from `jest-cli` though

Reply:
Or use something which normalizes the package data.

```js
require('read-pkg-up').sync({ cwd: require.resolve('jest') }).pkg.bin
{ jest: './bin/jest.js' }
```